cmake/modules/Hexagon.cmake

```diff
@@ -238,7 +246,8 @@ if(USE_HEXAGON_RPC)
     list(APPEND RUNTIME_HEXAGON_SRCS
       "${TVMRT_SOURCE_DIR}/hexagon/rpc/hexagon_rpc_stub.c"
     )
-    target_link_libraries(tvm_runtime PRIVATE cdsprpc)
+    target_link_libraries(tvm PRIVATE dl log ${CDSPRPC_LIBRARY})
```

Comment:
Was there a reason for adding in linking libtvm.so here? In some of the builds we only build libtvm_runtime.so, e.g. for Android and also for Hexagon device code, so libtvm.so can't be linked, but we still want custom runtime logging.

Reply:
The reason was so that I could add linked libraries after the targets were created. And I needed the targets to be created before this so that I could properly add definitions (in this case `USE_HEXAGON_LOGGING=1`) so that they would exist only for the target and would not pollute the global namespace.

Reply:
I don't understand how this answers the question...  Why did you add "target_link_libraries" for "tvm" here?  `libtvm.so` will never be built for Android.  Is it just for completeness in case it's added at some point in the future?